src/datepicker/datepicker-tools.ts

```diff
@@ -120,6 +123,8 @@ export function buildMonth(calendar: NgbCalendar, date: NgbDate, state: Datepick
       const newDate = new NgbDate(date.year, date.month, date.day);
       const nextDate = calendar.getNext(newDate);
 
+      const ariaLabel = i18n.getDayAriaLabel(newDate);
```

Comment:
As I mentioned in the previous review, could you please add a couple of tests to the `detepicker-service.spec.ts` that will test that labels on the days?

You would have caught unused `jsDate` bug the service if you had one...